src/backup_restore.cpp

```diff
@@ -5,6 +5,7 @@
 #include "logger.hpp"
 #include "parser.hpp"
 
+#include <utility/generic_utility.hpp>
```

Comment:
This is the reason why indirect inclusion should be avoided. 
Ideally this file should not have failed as it does not depend on the change, but it file failed because it was depending upon "generic_utility.hpp" but was getting it via "json_utility.hpp".